pkg/models/resources/v1alpha3/user/users.go

```diff
@@ -107,6 +108,10 @@ func (d *usersGetter) filter(object runtime.Object, filter query.Filter) bool {
 	switch filter.Field {
 	case iamv1alpha2.FieldEmail:
 		return user.Spec.Email == string(filter.Value)
+	case iamv1alpha2.InGroup:
```

Comment:
What happened if `user.spec.group` has duplicate content.

Reply:
There's no duplication here, The group name is unique in the cluster.